BUILDING.md

```diff
@@ -37,15 +33,73 @@ Building: Docker >> alpine-linux-x64 >> NODE_ENABLED
 Override build-steps ? (leave empty to run pre-configured steps): j2v8
```

Comment:
Is `j2v8` a build step? Or does it stand for `all`? I thought there were only these NodeJS --> CMake --> JNI --> Optimize --> Java/Android --> JUnit (didn't look up script names, a bit short on time, sorry)